examples/http/package.json

```diff
@@ -21,7 +21,6 @@
   },
   "dependencies": {
     "@opencensus/exporter-zipkin": "^0.0.9",
-    "@opencensus/instrumentation-http": "^0.0.9",
```

Comment:
Out of curiosity, why is this no longer needed?

Reply:
There are two ways to instrument supported plugins. 1) Manual approach: include respective dependency and use ```plugin.enable(...)``` interface. 2) Allow the library to instrument plugins automatically, for this start the trace instance as the first module in your application.

The changed the HTTP example to work with second (I think recommended) approach.